modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java

```diff
@@ -101,14 +103,63 @@ public List<ObjectField> getAccountEntryRelationshipObjectFields()
 		);
 	}
 
-	public List<KeyValuePair> getKeyValuePairs() {
-		List<KeyValuePair> keyValuePairs = new ArrayList<>();
+	public List<ObjectField> getNonrelationshipObjectFields() {
```

Comment:
This method is never used. You can remove it, since its code already has been putting in `getNonrelationshipObjectFieldsInfo()`.